src/components/Nav/index.js

```diff
@@ -26,61 +36,14 @@ function Nav() {
                 </CloseButton>
 
                 <MenuList>
-                    <MenuItem>Categories</MenuItem>
+                    <MenuTittle>Categories</MenuTittle>
```

Comment:
Only li are allowed to as direct children of ul. While tittle sounds cute I think you meant title ;)